### PR TITLE
Added image dump acceptance tests for dd, iso, E01, VHD, VMDK

### PR DESCRIPTION
**Pull request by uckelman-sf**
This PR provides an acceptance test harness for test images, as well as minimal test images for dd, iso, E01, VHD, and VMDK.

### CODE REVIEW

**Comment by simsong** [comment]
I just got a wacky message from Github when I clicked on the link it just sent (see below); any clue?


image

**Comment by uckelman-sf** [comment]
@simsong  That's likely due to the branch having changed due to rebasing.

File: appveyor.yml [hunk 1]
```diff
@@ -75,5 +78,8 @@ for:
 
     build_script:
       - ./bootstrap
-      - ./configure -q
-      - make -s
+      - ./configure
+      - make -j
```
Comment (simsong) [inline appveyor.yml:82]: really `-j`? Will that blow up the AppVeyor VM?

Comment (simsong) [reply appveyor.yml:82]: I guess there is only one way to find out...

**Review by simsong — APPROVED** [review]

File: appveyor.yml (hunk 1 above)
Comment (uckelman-sf) [reply appveyor.yml:82]: It worked. I won't wait for serial `make` unless there's no alternative.

**Comment by uckelman-sf** [comment]
@simsong This is ready for review.

**Pull request merged**